### PR TITLE
fix: TestExternalIPsServer_NoFile is not stable

### PR DESCRIPTION
**Pull request by denis-tingajkin**
Signed-off-by: denis-tingajkin <denis.tingajkin@xored.com>

https://github.com/networkservicemesh/sdk/issues/532

### CODE REVIEW

File: pkg/networkservice/common/externalips/server_test.go [hunk 1]
```diff
@@ -59,8 +59,17 @@ func TestExternalIPsServer_NoFile(t *testing.T) {
...
-				externalips.ToInternal(ctx, externalIP).Equal(internalIP)
+			actualExternal := externalips.FromInternal(ctx, internalIP)
+			if actualExternal == nil {
+				return
+			}
+			actualInternal := externalips.ToInternal(ctx, externalIP)
+			if actualInternal == nil {
+				return
+			}
+			require.NotNil(t, actualInternal)
```
Comment (Bolodya1997) [inline pkg/networkservice/common/externalips/server_test.go:70]: How it can be `nil`?

File: pkg/tools/fs/utils.go [hunk 2]
```diff
@@ -92,14 +92,13 @@ func monitorFile(ctx context.Context, filePath string, watcher *fsnotify.Watcher
 			if !strings.HasSuffix(filePath, filepath.Clean(e.Name)) {
 				continue
 			}
-			switch e.Op {
-			case fsnotify.Chmod:
-				continue
-			case fsnotify.Remove, fsnotify.Rename:
+			if e.Op&fsnotify.Remove+e.Op&fsnotify.Rename > 0 {
```
Comment (Bolodya1997) [inline pkg/tools/fs/utils.go:95]:
```go
e.Op&(fsnotify.Remove|fsnotify.Rename) != 0
```

File: pkg/tools/fs/utils.go [hunk 3]
```diff
@@ -92,14 +92,13 @@ func monitorFile(ctx context.Context, filePath string, watcher *fsnotify.Watcher
...
 				logger.Warn("Removed")
 				if !sendOrClose(ctx, notifyCh, nil) {
 					return
 				}
+			} else if e.Op&fsnotify.Write == 0 && e.Op&fsnotify.Create == 0 {
```
Comment (Bolodya1997) [inline pkg/tools/fs/utils.go:100]:
```go
e.Op&(fsnotify.Write|fsnotify.Create) == 0
```

File: pkg/networkservice/common/externalips/server_test.go (hunk 1 above)
Comment (denis-tingajkin) [reply pkg/networkservice/common/externalips/server_test.go]: Thanks, fixed

**Review by haiodo — APPROVED** [review]

**Pull request merged**